recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = "2.38.0" %}
+{% set win_version = version[:-2] %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz  # [unix]
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz  # [unix]
```

Comment:
As a proposal, we could try using the commit for better stability. If you want, this can be added to the jinja header.

https://github.com/ellson/graphviz/archive/f54ac2c9313ae80ccf76ef4ac6aa9be820a23126.zip